src/components/AddBook.js

```diff
@@ -19,7 +13,14 @@ const AddBook = () => {
       <form>
         <input type="text" id="title" placeholder="Book Title" value={title} onChange={(e) => setTitle(e.target.value)} />
         <input type="text" id="author" placeholder="Author" value={author} onChange={(e) => setAuthor(e.target.value)} />
-        <button type="button" onClick={() => dispatch(addBook(getData(title, author)))}>
+        <button
+          type="button"
+          onClick={() => dispatch(sendBook(
+            {
+              item_id: uuidv4(), title, author, category: 'fiction',
+            },
+          ))}
+        >
```

Comment:
- [ ] [OPTIONAL] it would be better to clean the input after creating a book for good user experience 
<img width="1342" alt="Screenshot 2023-05-11 at 19 30 16" src="https://github.com/ehmaddd/bookstore/assets/61790587/5b8e0480-10fc-473f-b2c1-89520b88370c">